### ISSUE TITLE
Commit to show PR# owner instead of committer/reviewer

### ISSUE THREAD

**Issue by thajith** [screwdriver-cd/screwdriver#1407]
**What happened**:
When a PR# is merged , we are seeing the Merged individual name next to the commit instead of the PR# owner as part of the Git Repo.
Find the attached screen shot for the reference of the location.

**What you expected to happen**:
Would like to see the PR# owners name next to the commit.

**How to reproduce it**:
Create a PR# (created by Dev1) , merge to be done by dev2. Screen shows Dev2 againist the commit instead of dev1.

**Comment by jithin1987** [comment]
This is debatable. Will have to look deeper into how it fits into our Data model. What SD is processing is a commit event fro Git for the branch of the user created pipeline.

**Comment by jithin1987** [comment]
We should take a page from Git and display both info

<img width="1011" alt="screen shot 2019-02-05 at 12 01 17 pm" src="https://user-images.githubusercontent.com/193126/52300847-c8caf980-293d-11e9-9a34-17dcb0754e83.png">


cc @DekusDenial

**Comment by tkyi** [comment]
Seems like we are already trying to differentiate between "author" and "committer" in the backend, but the "author" always shows up as the "committer"

Event data:
```
...
    "commit": {
        "author": {
            "avatar": "https://github.com/avatars/u/497?",
            "name": "Dao Lam",
            "username": "d2lam",
            "url": "https://github.com/d2lam"
        },
        "message": "small fix (#397)",
        "url": "https://github.com/screwdriver-cd/screwdriver/commit/4a99b46488a2d605156285469007ea43becc5689"
    },
    "createTime": "2019-03-12T10:59:48.542Z",
    "creator": {
        "avatar": "https://github.com/avatars/u/497?",
        "name": "Dao Lam",
        "username": "d2lam",
        "url": "https://github.com/d2lam"
    },
...
```

Code where we get "creator"/"committer" data:
https://github.com/screwdriver-cd/models/blob/master/lib/eventFactory.js#L466-L471
Code where we get "author" data:
https://github.com/screwdriver-cd/models/blob/master/lib/eventFactory.js#L473-L481
https://github.com/screwdriver-cd/scm-github/blob/master/index.js#L860-L873
https://developer.github.com/v3/repos/commits/#get-a-single-commit (Github API response)

**Comment by tkyi** [comment]
Should be resolved
<img width="446" alt="Screen Shot 2022-10-12 at 4 42 01 PM" src="https://user-images.githubusercontent.com/3230529/195467491-9578db4c-2355-4e26-8ab0-6260031167fe.png">
